influxql/select_test.go

```diff
@@ -2645,6 +2645,104 @@ func TestSelect_Elapsed_Boolean(t *testing.T) {
 	}
 }
 
+func CheckPoints(t *testing.T, ic *IteratorCreator, query string, expected [][]influxql.Point) {
```

Comment:
Yea, if you can remove this that would be great. I understand the desire and I would consider a PR that changes everything for this pattern, but I don't want to mix test styles at the moment. When I've tried that in the past, it became unruly and I had to undo all of it.